tests/comfy_cli/command/test_model_download_background.py

```
@@ -1110,6 +1112,702 @@ def write_then_race(ws, state):
+            models._download_worker(state_file=str(path))
+
+        assert not claim.exists()
+
+
+class TestClaimAtomicPublish:
+    """`acquire_claim` writes the payload to a private temp name and publishes
+    it with `os.link`: creation is still the atomic decider, but the claim is
+    never visible half-written — a colliding submitter that reads it can never
+    mistake a live winner mid-write for a stale claim and unlink it."""
+
+    def test_short_writes_still_publish_a_complete_payload(self, workspace, monkeypatch):
```

Pylint (4.0.7)

[convention] Missing function or method docstring

(C0116)